src/app/[locale]/incident/add/components/questions/TextAreaInput.test.tsx

```diff
@@ -0,0 +1,181 @@
+import { test, expect } from 'vitest'
+import { render, screen } from '@testing-library/react'
+import { FieldTypes } from '@/types/form'
+import { FormMock } from '../../../../../../../__mocks__/FormMock'
+import { FieldValues } from 'react-hook-form'
+import { TextAreaInput } from '@/app/[locale]/incident/add/components/questions/TextAreaInput'
+
+const renderWithForm = (
+  component: React.ReactElement,
+  defaultValues: FieldValues
+) => {
+  return render(<FormMock defaultValues={defaultValues}>{component}</FormMock>)
+}
+
+// Visiblity tests
+test('should show textarea input', async () => {
+  const field = {
+    key: 'Vuurwerk_overlast_simpel',
+    field_type: FieldTypes.TEXT_INPUT,
+    meta: {
+      label: 'Weet u het nummer van de container?',
+    },
+    required: false,
+  }
+
+  renderWithForm(<TextAreaInput field={field} />, {})
+
+  expect(screen.getByRole('textbox')).toBeInTheDocument()
```

Comment:
Heel goed idee om `getByRole` te gebruiken waar mogelijk! 💯

Wist je dat je ook kunt checken of de toegankelijke naam goed is ingesteld? Goeie manier om gelijk in unit tests de accessibility te checken. Scheelt een hoop fixes achteraf!

```tsx
screen.getByRole('textbox', { name: field.meta.label })
```